observability/logging/cluster-logging-deploying.adoc

```diff
@@ -7,57 +7,53 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
-[id="prerequisites_cluster-logging-deploying"]
-== Prerequisites
-* Ensure that you have downloaded the {cluster-manager-url-pull} as shown in _Obtaining the installation program_ in the installation documentation for your platform.
-+
-If you have the pull secret, add the `redhat-operators` catalog to the OperatorHub custom resource (CR) as shown in _Configuring {product-title} to use Red Hat Operators_.
-endif::[]
-
-//Installing the CLO via webconsole
-include::modules/cluster-logging-deploy-console.adoc[leveloffset=+1]
-include::modules/create-cluster-logging-cr-console.adoc[leveloffset=+1]
-
-// Install using CLI
-include::modules/cluster-logging-deploy-cli.adoc[leveloffset=+1]
-include::modules/create-cluster-logging-cli.adoc[leveloffset=+1]
+You deploy {logging} by installing the {loki-op} or {es-op} to manage your log store, followed by the {clo} to manage the components of logging. You can use either the {product-title} web console or the {product-title} CLI to install or configure {logging}.
 
-[id="cluster-logging-deploying-postinstallation"]
-== Postinstallation tasks
-
-After you have installed the {clo}, you can configure your deployment by creating and modifying a `ClusterLogging` custom resource (CR).
+include::snippets/logging-elastic-dep-snip.adoc[leveloffset=+1]
 
 [TIP]
 ====
-If you are not using the Elasticsearch log store, you can remove the internal Elasticsearch `logStore` and Kibana `visualization` components from the `ClusterLogging` custom resource (CR). Removing these components is optional but saves resources. See xref:../../observability/logging/log_storage/logging-config-es-store.adoc#cluster-logging-removing-unused-components-if-no-elasticsearch_logging-config-es-store[Removing unused components if you do not use the Elasticsearch log store].
+You can alternatively apply all example objects.
 ====
 
-include::modules/cluster-logging-about-crd.adoc[leveloffset=+2]
-include::modules/configuring-log-storage-cr.adoc[leveloffset=+2]
-include::modules/configuring-logging-collector.adoc[leveloffset=+2]
-include::modules/configuring-log-visualizer.adoc[leveloffset=+2]
-include::modules/cluster-logging-deploy-multitenant.adoc[leveloffset=+2]
+include::modules/logging-es-deploy-console.adoc[leveloffset=+1]
+
+include::modules/logging-es-deploy-cli.adoc[leveloffset=+1]
+
+--
+include::snippets/logging-retention-period-snip.adoc[leveloffset=+1]
+--
+
+include::modules/logging-loki-cli-install.adoc[leveloffset=+1]
+
+include::modules/logging-loki-gui-install.adoc[leveloffset=+1]
+
+ifdef::openshift-origin[]
+[id="prerequisites_cluster-logging-deploying"]
+== Prerequisites
+* Ensure that you have downloaded the {cluster-manager-url-pull} as shown in _Obtaining the installation program_ in the installation documentation for your platform.
++
+If you have the pull secret, add the `redhat-operators` catalog to the OperatorHub custom resource (CR) as shown in _Configuring {product-title} to use Red{nbsp}Hat Operators_.
+endif::[]
```

Comment:
I don't have a preview for OKD, but wouldn't this resolve as an assembly level prereqs section that comes _after_ all the modules and right before the additional resources section? If you'd need further SME review to move this then I guess you can leave it, but I'm guessing you can probably just move it to before the modules?